lib/tasks/deploy.rake

```diff
@@ -48,7 +48,14 @@ namespace :deploy do
     sh "docker build -t #{tag} ."
     sh "docker push #{tag}"
 
-    trigger_deployment(application, build, branch) if branch == 'staging'
+    next unless branch == 'staging'
+
+    deployed_apps = [application]
+    if ENV.key?('DEPLOY_APPS')
+      deployed_apps.concat(ENV.fetch('DEPLOY_APPS').split(','))
```

Comment:
maybe rename the env key to `EXTRA_DEPLOY_APPS`? Or not include the default app name?

Because running `rake deploy DEPLOY_APPS=something-else` looks like it would not deploy to the default target.